### ISSUE TITLE
what version of ubuntu to use with intel-creatvive

### ISSUE THREAD

**Issue by shomedas** [OpenGP/htrack#25]
I tried using intel-creative realsense with your code on ubuntu 14.04.1 LTS. It gave message:

FATAL: Intel Real sense Not available on your OS.

What version of Ubuntu do I have to use for intel realsense.

**Comment by ataiya** [comment]
This falls in the category of "I'd removed legacy code so people don't try run things that are clearly not marked as supported". Anyway, we tried to port the code to windows, where the realsense sensor class was using RealSenseSDK, but GpGPU performance was terrible, so I just stuck to linux.

On linux there is the new librealsense which I integrated, but it's in the new version of the tracker that will be published in a few months. (not making the mistake above again... :P)
